### PR TITLE
T: Fix pretty-printers tests on 2023.1

### LINKED ISSUE #10009: Pretty printer tests don't work with 2023.1 builds
```
Error while running LLDB:
Traceback (most recent call last):
  File "/home/runner/work/intellij-rust/intellij-rust/pretty_printers_tests/./lldb_batchmode.py", line 197, in <module>
    debugger = lldb.SBDebugger.Create()
AttributeError: module 'lldb' has no attribute 'SBDebugger'
```

### PR DESCRIPTION
**Pull request by ortem**
Fixes #10009

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -611,13 +611,27 @@ project(":ml-completion") {
...
-    if (platformVersion >= 231) return@task
     doLast {
+        val hostPlatform = DefaultNativePlatform.host()
+        val archName = if (hostPlatform.architecture.isAmd64) {
+            "x64"
+        } else if (hostPlatform.architecture.isArm) {
+            "aarch64"
+        } else {
+            error("Unsupported architecture")
+        }
```
Comment (Undin) [inline build.gradle.kts:622]: nit: `when` maybe?

File: build.gradle.kts [hunk 2]
```diff
@@ -637,13 +649,7 @@ task("runPrettyPrintersTests") {
             "cmd /C set PYTHONIOENCODING=utf8 & $runCommand".execute("pretty_printers_tests")
         } else {
             // TODO: Remove after CLion snapshot builds provide these files with required permissions
```
Comment (Undin) [inline build.gradle.kts:651]: It seems this comment should be removed as well

**Review by Undin — APPROVED** [review]

File: build.gradle.kts (hunk 2 above)
Comment (ortem) [reply build.gradle.kts:651]: It's still relevant because we still have to invoke `setExecutable` on macOS

Comment (Undin) [reply build.gradle.kts:651]: Oh, I see

**Pull request merged**